doc/api/vm.md

```diff
@@ -334,10 +390,10 @@ let code =
  vm.runInThisContext(code)(require);
  ```   
 
-_Note: `require()` in the above case shares the state with context it is passed
-from. This might introduce risks when unknown code is executed, e.g. altering
-objects from the calling thread's context in unwanted ways. It is advisable to
-run `vm` code in a separate process._
+*Note*: The `require()` in the above case shares the state with context it is
+passed from. This may introduce risks when untrusted code is executed, e.g.
+altering objects from the calling thread's context in unwanted ways. It is
+advisable to always run `vm` code within in a separate process.
```

Comment:
+1 … I actually have been given this advice directly by core collaborators, presumably based on reading this paragraph.